### DISCUSSION TITLE
[Feature] Widget: Hide message if nothing to display

### DISCUSSION THREAD

**Discussion by xcy7e** [TechbeeAT/jtxBoard#147] [Polls]
Would be cool to have the possibility to hide the message "nothing to show by given filter" (or however this translates into English) on the widget.

I really like the possibility of showing my tasks due on today on my home screen. When there is nothing to do, it's obviously empty, so I do not really need that message, because it rather disturbes the theme.

On the other hand, I agree, giving no feedback whenever a filter is in play is rarely a good choice but regarding this as a widget, coming handy on various types of user screens and themes this would be really nice, also regarding most of your users might - to this day - be experienced users, I guess.

What do you think about it?

PS: I do not want to ask for too much, so just as a side note, a more decent font color of the widget would be nice, like gray or black, or even white, because the light blue is very special.

PPS: Great work with that opacity settings that's awesome.

**Comment by patrickunterwegs** [comment]
Hey @xcy7e 
Well, in that case I'm not so keen on changing it 😉 
I definitely want to keep the message, I really don't want to get any support requests why the widget stays empty, so I'd rather give some feedback there 😉  And well, I wouldn't overestimate the users' experience...

I was just playing around with he opacity of the message, but it wouldn't change a lot, it would just get harder to read...

About the colours: Those are based on a color concept by Google/Material 3: 
https://m3.material.io/styles/color/dynamic-color/overview

From Android 12 you can change the colours based on your background, jtx Board will take those dynamic colours and apply them accordingly. The colour scheme not only defines the background colours, but also the colours ON the backgrounds and surfaces, so the system would actually generate/calculate a colour that fits best according to the colour scheme. In case of the widget, the background color is the "primaryContainer", the text on that container is the derived "onPrimaryContainer" given by the colour scheme. 
So theoretically I could choose another colour there, but this comes with the downside, that the colour would not fit to the colour scheme anymore, dynamic colours might not be applied, day and night color has to be defined separately, ...

So I'm afraid in this case I would have to reject your proposal, but I hope you understand after my explanations 😉

**Reply by patrickunterwegs** [reply]
@xcy7e , in order to give others the chance to follow up on your proposal, I transformed it into a discussion, if many people have the same opinion as you, I'd reconsider it 😉
